Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -26,6 +26,9 @@ Neos:
 
       # Defines the PSR-15 middlewares that a request goes through in a flow
       middlewares:
+        'session':
+          position: '10'
```

Comment:
Let's change this to `after trustedProxies` after rebasing onto #2153 - once we get e.g. FlashMessages done, that will need to be moved in between or even before trustedProxies.

Reply:
btw., that's the reason the tests currently fail:
`flashMessagesGetRenderedAfterRedirect` - we need to move the flashmessages component first